### DISCUSSION TITLE
I am unable to sign up for Azure for students with GitHub

### DISCUSSION THREAD

**Discussion by nandakishore1807** [community/community#45978] [GitHub Education]
I am unable to sign up for Azure for students with GitHub it is showing your are not eligible for free account 
What i want to do?
Can anyone help me
![Screenshot_20230201_093248](https://user-images.githubusercontent.com/77160824/215942717-f0fa9370-949d-400f-9c38-002590e2cea0.jpg)

**Comment by jge162** [comment]
You might need to reach out to Azure via support. Hard to say why your are declined.

**Reply by nandakishore1807** [reply]
How to contact the Azure support

**Reply by jge162** [reply]
Here you go ->[Azure Support](https://azure.microsoft.com/en-us/support/create-ticket/)
